IdentityCore/src/cache/ios/MSIDKeychainTokenCache.m

```diff
@@ -381,6 +381,59 @@ - (BOOL)saveAppMetadata:(MSIDAppMetadataCacheItem *)item
     return appMetadataitems;
 }
 
+- (BOOL)saveOrUpdateMetadataItem:(NSData *)item
```

Comment:
let's follow convention and pass:
1. cacheitem
2. cache key
3. serializer

Reply:
Done,